db/rebuildSubjectHierarchy.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2016, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * db/rebuildSubjectHierarchy.js
+ */
+const utils = require('./utils.js');
+
+utils.doImport()
+.then(() => {
+  utils.seq.models.Subject.rebuildHierarchy()
+  .then(() => {
+    console.log('Successfully rebuild Subject Hierarchy.'); // eslint-disable-line no-console
```

Comment:
since this is printed upon completion it should be in past tense ("rebuilt" rather than "rebuild")

Reply:
True